oak/server/rust/oak_runtime/src/config.rs

```diff
@@ -0,0 +1,62 @@
+//
+// Copyright 2020 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+use crate::proto::manager::{
+    ApplicationConfiguration, LogConfiguration, NodeConfiguration, WebAssemblyConfiguration,
+};
+
+/// Create an application configuration.
+///
+/// - module_name_wasm: Node name and Wasm bytes.
+/// - logger_name: Node name to use for a logger configuration; if empty no logger will be included.
+/// - initial_node: Initial node to run on launch.
+/// - entrypoint: Entrypoint in the initial node to run on launch.
+pub fn application_configuration(
+    module_name_wasm: Vec<(String, Vec<u8>)>,
```

Comment:
I think a `HashMap` would be more idiomatic here (and you should be able to iterate over its entries in exactly the same way).

Reply:
True, but this PR is just moving code around so deferring for now.